packages/app/src/App.tsx

```diff
@@ -94,6 +94,7 @@ import { techDocsPage } from './components/techdocs/TechDocsPage';
 import { ApacheAirflowPage } from '@backstage/plugin-apache-airflow';
```

Comment:
There would be no changes in this file for this PR, it's just an example to try it out.